lib/core.sh

```diff
@@ -396,13 +401,10 @@ function build_image_env(){
     sudo bash -c "echo 'export PATH=/opt/yaourt/bin:\$PATH' > ${maindir}/root/etc/profile.d/${CMD}.sh"
     sudo chmod +x ${maindir}/root/etc/profile.d/${CMD}.sh
 
-    info "Install ${NAME} script..."
-    sudo pacman --noconfirm --root ${maindir}/root -S git
-    _install_from_aur ${maindir} "${CMD}-git" "${CMD}.install"
-    sudo pacman --noconfirm --root ${maindir}/root -Rsn git
-
     info "Setting up the pacman keyring (this might take a while!)..."
-    sudo arch-chroot ${maindir}/root bash -c "pacman-key --init; pacman-key --populate archlinux"
+    sudo pacman --noconfirm --root ${maindir}/root -S psmisc
+    sudo ${maindir}/root/opt/junest/bin/jchroot ${maindir}/root bash -c "pacman-key --init; pacman-key --populate archlinux; killall gpg-agent"
```

Comment:
Ok, I've got it! This is how we can gracefully shutdown the gpg agent owned by a specific junest sandbox:

```
$ echo "KILLAGENT" | gpg-connect-agent -S /etc/pacman.d/gnupg/S.gpg-agent -r /dev/stdin
```